Dockerfile

```diff
@@ -3,16 +3,18 @@ RUN apt-get update
 RUN pip install "poetry==1.0.0"
 WORKDIR /do-me
 COPY poetry.lock poetry.toml pyproject.toml /do-me/
-RUN poetry install
+RUN poetry config virtualenvs.create false --local && poetry install
 COPY ./todo_app/ /do-me/todo_app/
 EXPOSE 5000
 
 FROM base as production
 COPY ./run_gunicorn.sh ./run_gunicorn.sh
+RUN chmod +x ./run_gunicorn.sh
```

Comment:
It's currently failing to run on Heroku. https://jackiemaw-do-me.herokuapp.com/

I think the issue is just Windows line endings in your `run_gunicorn.sh` file. 

You could edit your git settings to push LF line endings with something like `git config --global core.autocrlf=input`

Or you could edit the file in VSCode. Click on CRLF in the bottom right of the screen when viewing the file and switch it to LF. Add, commit and push the change.

Reply:
thanks - that did the trick, all working now